README.md

```diff
@@ -10,7 +10,7 @@ to simplify building and publishing Kubernetes APIs from scratch.
 
 ### Installalation 
 
-It is strongly recommended that you use a released version. Release binaries are available on the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page.
+It is strongly recommended that you use a released version. Release binaries are available on the [releases](https://sigs.k8s.io/kubebuilder/releases) page.
```

Comment:
No need to change this. This link is a 404
`https://sigs.k8s.io/kubebuilder/releases`